.gitignore

```diff
@@ -1,3 +1,4 @@
 vendor/
 composer.lock
-.DS_Store
\ No newline at end of file
+.DS_Store
+.idea
```

Comment:
these should not be in the project gitignore, but rather editor global .gitignore – could we remove both?